app/assets/stylesheets/application.scss

```diff
@@ -89,18 +89,21 @@ body {
 .form-signin .btn {
   font-size: 80%;
   border-radius: 5rem;
-  letter-spacing: .1rem;
+  letter-spacing: 0.1rem;
   font-weight: bold;
   padding: 1rem;
   transition: all 0.2s;
 }
 
-.big-checkbox {width: 20px; height: 20px;}
+.big-checkbox {
+  width: 20px;
+  height: 20px;
+}
```

Comment:
I think my text editor did a bunch of auto-formatting when I opened this file...I definitely did not mean for all of the changes to occur.

Reply:
no problemo, I appreciate perfect formatting!